docs/user_guide/workloads/run_mpi.md

```diff
@@ -0,0 +1,106 @@
+---
+id: run_mpi
+title: Run MPI Jobs
+description: How to run MPI jobs with YuniKorn
+keywords:
+ - mpi
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+
+# MPI Operator
+
+For more details, please read https://github.com/kubeflow/mpi-operator#installation for deploying this operator.
```

Comment:
Change https://github.com/kubeflow/mpi-operator#installation to
[Installation steps of mpi-operator](https://github.com/kubeflow/mpi-operator#installation)

Reply:
Applied both suggestions!